quickemu.1.md

```diff
@@ -134,57 +137,34 @@ See this (old) video where I explain some of my motivations for creating Quickem
 
 ## Requirements
 
-  * [QEMU](https://www.qemu.org/) (*6.0.0 or newer*) **with GTK, SDL, SPICE & VirtFS support**
-  * [bash](https://www.gnu.org/software/bash/) (*4.0 or newer*)
-  * [Coreutils](https://www.gnu.org/software/coreutils/)
-  * [EDK II](https://github.com/tianocore/edk2)
-  * [grep](https://www.gnu.org/software/grep/)
-  * [jq](https://stedolan.github.io/jq/)
-  * [LSB](https://wiki.linuxfoundation.org/lsb/start)
-  * [procps](https://gitlab.com/procps-ng/procps)
-  * [python3](https://www.python.org/)
-  * [macrecovery](https://github.com/acidanthera/OpenCorePkg/tree/master/Utilities/macrecovery)
-  * [mkisofs](http://cdrtools.sourceforge.net/private/cdrecord.html)
-  * [usbutils](https://github.com/gregkh/usbutils)
-  * [util-linux](https://github.com/karelzak/util-linux)
-  * [sed](https://www.gnu.org/software/sed/)
-  * [spicy](https://gitlab.freedesktop.org/spice/spice-gtk)
-  * [swtpm](https://github.com/stefanberger/swtpm)
-  * [Wget](https://www.gnu.org/software/wget/)
-  * [xdg-user-dirs](https://www.freedesktop.org/wiki/Software/xdg-user-dirs/)
-  * [xrandr](https://gitlab.freedesktop.org/xorg/app/xrandr)
-  * [zsync](http://zsync.moria.org.uk/)
-
-# Install Quickemu
```

Comment:
If they have the man page surely they have already installed so not needed in the man page(s)